### DISCUSSION TITLE
bind label background color to a dict?

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1578] [Q&A]
I want to change label backgroud color dynamically and bind the color to a dict

how can I achive such effect,anyone has some thouthgs?

![微信截图_20230908013619](https://github.com/zauberzeug/nicegui/assets/25096045/4ae7bc13-ada3-4e00-95b9-d394162a0d52)
items in the picture could dynamically change backgroud color when the value it bind to changed.

**Comment by rodja** [comment]
What have you tried already? Have you read the documentation about binding?

**Reply by v479038280** [reply]
I tried this way #1569 ,  I read the documentation and the code about about binding ,and tried to creat such element by myself ,It seems a little difficult for me . So I asked if there are other way to achieve this

**Reply by rodja** [reply]
So this is the same question as #1569? Why repost it here again instead of clarifying your problem over there? Its quite time consuming to read and manage all this conversations...

**Reply by v479038280** [reply]
I'm still trying to build a new element to change the background .
This is another way to achive this ,  #1569 could achieve that effect with only one item.but when adding several items ,there is some problem,

**Reply by rodja** [reply]
> This is another way to achive this

Sorry. I don't understand. What do you mean by "this"? I do not see "another way" in what has been written here. What exactly is the problem?

**Reply by v479038280** [reply]
#1569 is solved . Here is the problem,
I add some labels in the card, e.g, Label A,B,C,D,E, and  I want to bind the label backgound color to dict` V={'a':True,'b':False,'c':False,'d':True,'e':False}`,   When the value is True ,the color is green, and when False ,the color become red

If I could creat such a element  `ui.mylabel('A').bind_bgcolor_from(V,'a')`, or if there are other ways to achieve such effect
![微信截图_20230908023114](https://github.com/zauberzeug/nicegui/assets/25096045/eefae2d2-2f40-47ce-82ee-9f5f5f93279c)

**Reply by rodja** [reply]
Ah, now it's more clear. Thanks. I've created a new example on how to achieve something like that in #1581.
